cmd/manager/main.go

```diff
@@ -25,6 +25,8 @@ import (
 	nginxApis "github.com/tsuru/nginx-operator/pkg/apis"
 	v1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/util/intstr"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
```

Comment:
The same here. This pkgs are already imported in the `_ "k8s.io/client-go/plugin/pkg/client/auth"` above.

Reply:
Nice, I removed them both.